ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/vxlan_interface.py

```diff
@@ -67,6 +67,17 @@ def vxlan_interface(self) -> dict | None:
                         # This is legacy behavior so we will leave stricter enforcement to the schema
                         vrfs[key] = {"vni": vni}
 
+                        if get(vrf, "_l3_multicast_enabled"):
```

Comment:
the logic below is ip_addressing logic, which should be implemented in the ip_addressing class. It will make it simpler to modify to other algorithms later.

Reply:
nevermind. We will move it later together with l2 multicast